### PR TITLE
Mark move_base headers for installation

### PR DESCRIPTION
**Pull request by garyservin**
This is needed to use move_base on Android either with rosjava using JNI or as a native activity.

Do you think an `if(ANDROID)` is necessary around this part?

### CODE REVIEW

File: move_base/CMakeLists.txt [hunk 1]
```diff
@@ -70,3 +70,10 @@ install(
     LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
     RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
+
+## Mark cpp header files for installation
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.h"
+  PATTERN ".svn" EXCLUDE
```
Comment (mikeferguson) [inline move_base/CMakeLists.txt]: since we use git, not svn, this line should not be needed.

**Comment by mikeferguson** [comment]
Can you squash these two commits into one (cleaner history)

**Comment by garyservin** [comment]
Done! thanks for the help!

**Comment by mikeferguson** [comment]
Thanks for the contributions -- I'll merge this as soon as the currently running devel job completes.

**Pull request merged**